tree/dataframe/inc/ROOT/RDF/RDefine.hxx

```diff
@@ -119,6 +119,8 @@ public:
    void InitSlot(TTreeReader *r, unsigned int slot) final
    {
       if (!fIsInitialized[slot]) {
+         for (auto &define : fDefines.GetColumns())
+            define.second->InitSlot(r, slot);
```

Comment:
Could the `InitSlot` call still be done from an action or Filter too as it was before? Is that a problem?

Reply:
sure, and it should not be a problem